resources/definitions/mixware_wand.def.json

```diff
@@ -0,0 +1,194 @@
+{
+  "name": "Wand",
+  "version": 2,
+  "inherits": "fdmprinter",
+  "metadata": {
+    "visible": true,
+    "author": "Mixware",
+    "manufacturer": "Mixware",
+    "file_formats": "text/x-gcode",
+    "has_materials": true,
+    "has_machine_quality": true,
+    "preferred_material": "mixware_pla",
+    "preferred_quality_type": "coarse",
+    "has_variants": false,
+    "platform": "mixware_wand_platform.STL",
+    "machine_extruder_trains": {
+      "0": "mixware_wand_extruder_0"
+    }
+  },
+  "overrides": {
+    "machine_name": {
+      "default_value": "Wand"
+    },
+    "machine_start_gcode": {
+      "default_value": "G28 ;Home\n;Prime the extruder\nM117 Print\nG92 E0\nG1 F1500 X-2 Y0 Z0.28\nG1 F2400 X-2 Y70 Z0.28 E8\nG1 F1500 X-1.7 Y70 Z0.28\nG1 F2400 X-1.7 Y0 Z0.28 E16\nG92 E0\nG92 E0\nG1 Z0.18 F2400 ;Move Z Axis up"
+    },
+    "machine_end_gcode": {
+      "default_value": "M104 S0\nM140 S0\n;Retract the filament\nG92 E1\nG1 E3 F300\nG1 F1500 Z100\nG28 X0 Y0\nM25\nM84"
+    },
+    "material_diameter": {
+      "default_value": 1.75
+    },
```

Comment:
```suggestion
```
You already define this setting in the extruder (and that is also where it belongs!)